subprojects/gradle/sentry-config/src/main/kotlin/com/avito/android/sentry/ProjectExtensions.kt

```diff
@@ -72,3 +84,50 @@ private fun from(project: Project): SentryConfig {
         SentryConfig.Disabled
     }
 }
+
+/**
+ * Can't depend on GradleLoggerFactory, because of cyclic dependency:
+ *  - gradle logger depends on sentry to send criticals
+ */
+private class SimpleLoggerFactory : LoggerFactory {
+
+    override fun create(className: String): Logger {
+
+        val defaultHandler = DefaultLoggingHandler(destination = Slf4jDestination(className))
+
+        return DefaultLogger(
+            debugHandler = defaultHandler,
+            infoHandler = defaultHandler,
+            warningHandler = defaultHandler,
+            criticalHandler = defaultHandler
+        )
+    }
+}
+
+/**
+ * Can't reuse this class from gradle logger
+ * and can't move it to common logger, because it needs dependency on slf4j in that case
+ */
+private class Slf4jDestination(private val className: String) : LoggingDestination {
```

Comment:
So, I see a separate module but copy-paste

Reply:
so? explanation in comments is not enough ?

Reply:
I don't understand why we need this in common. We could create a module in `gradle` projects

Reply:
extracted https://github.com/avito-tech/avito-android/pull/695/commits/df487e1ba1be173b131b42ae67ef4898be63ed90